### ISSUE TITLE
tesseract-git build error

### ISSUE THREAD

**Issue by Hooperla** [m-ab-s/media-autobuild_suite#2185]
I'm probably doing something wrong but dragging and dropping the files onto the issue section doesn't seem to put them in correctly so I just decided to put them on a tiny MEGA upload instead (sorry for any inconvenience in advance).
MEGA Link: REMOVED (Logs now properly in a Github comment)

I admit that I don't know much of anything about how this program works or the programs that it fetches to do what it does. I just decided to try and build something since I'm on Windows and this seemed like the easiest way to do it (according to various posts I found at least). So, I downloaded the latest version from the Github page, put it in a root folder on my harddrive so it wouldn't have any spaces like it wanted, and just double-clicked.

I installed the 64-bit version only because I figured that would cover my needs.

I did Non-redistributable, because I'm just doing this for myself, and the rest I simply did yes when it seemed harmless or no when the description given made it seem buggy or more trouble than it was worth.
After going through all that it started downloading and I just let it sit so I didn't mess anything up, but then it stopped and told me that it had an error. 
It seems that "tesseract-git" wasn't working properly and the program stopped when it couldn't make it work.
I tried once again to the same result of it not working and then did the recommendation on the Github page to delete the folder and then tried once more, also leading to the same error.
I bundled up all three various logs just in case something between them would be useful or show some important difference.

Sorry that I can't be more descriptive like I would wish to be, but I just don't know enough about all the various aspects of the program to really explain it in a nuanced way.

PS: I feel that you should put on the Github page that when the program fails it sends information automatically. I'm not saying the developers are nefarious or anything (especially since the logs do indeed strip identifiers from what I can tell), but I take privacy seriously and would have liked a heads-up on that (or better yet an option to send the info or not when an error comes up and having an explanation from the devs on how it is stripped of identifiers and could help them out as an incentive to do so).

**Comment by 1480c1** [comment]
You should be able to attach zip files directly to github issues no?

Either way, right now testing some changes for prompting the user to upload logs or not to 0s0.st automatically, related https://github.com/m-ab-s/media-autobuild_suite/issues/2163

**Comment by Hooperla** [comment]
Log Bundles:
[logs (1st Try).zip](https://github.com/m-ab-s/media-autobuild_suite/files/8605708/logs.1st.Try.zip)
[logs (2nd Try).zip](https://github.com/m-ab-s/media-autobuild_suite/files/8605709/logs.2nd.Try.zip)
[logs (3rd Try after deleting build folder).zip](https://github.com/m-ab-s/media-autobuild_suite/files/8605711/logs.3rd.Try.after.deleting.build.folder.zip)

Huh, I swear it didn't work when I tried to do that earlier. Anyway, thanks for the response!

**Comment by Hooperla** [comment]
It seems that deleting "lib, opt, bin-global, bin-audio, bin-video in local## before building" as suggested by [LigH-de](https://github.com/LigH-de) cleared the issue of "tesseract-git" not working. 
Also, I have realized I was going completely about building what I intended the wrong way and have now really started trying to figure this building stuff out through MSYS2 proper and the like. Despite me using it incorrectly and mucking it up somehow with the building, I thank the creators of the program for their quick response to my issue and hope the logs can help future users in some regard. If nothing else the program made me want to learn more on my own and I'm grateful for that.
